src/physics/cam/nn_interface_cam.F90

```diff
@@ -123,8 +123,8 @@ subroutine nn_convection_flux_CAM(pres_cam, pres_int_cam, pres_sfc_cam, &
             !! precipitation at surface from one call to parameterisation
 
         ! Local input variables on the SAM grid
```

Comment:
Personally, I would find some info in the comments useful what these variables are. Or is that info available in documentation elsewhere?

Reply:
I have fleshed out the comments and documentation in https://github.com/m2lines/CAM-ML/pull/20/commits/498d6fdfc6ab7ef5365036b918a0ef0854188b56